assignment3

```diff
@@ -0,0 +1,57 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package assignment3;
+
+
+/**
+ *
+ * @author Home pc
+ */
+public class Assignment3 {
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+        // TODO code application logic here
+        String daniel = "daniel";
+        int stringSize = daniel.length();
+        for(int i=0; i < daniel.length();i++)
+        {
+            char letter = daniel.charAt(i);
```

Comment:
So at this point you have a `char` which is numeric in order to turn a character into a string I provided this method on the String object in the assignment instructions `String.valueOf()`. As we saw before if you do `char + char + char` you will get an `int`.

To create a string we need to concatenate strings. `String.valueOf()` will return a string for a numeric type like a character. Because `String + String + String` evaluates to a String.

The other trick you have to consider with your look is how to read the string backwards.